REACT_STYLEGUIDE.MD

```diff
@@ -0,0 +1,194 @@
+# React Style Guide
+
+### Prefer Stateless functional components where possible.
+Stateless function components are more concise, and there are plans for react to increase performance of them.
+Good:
+```
+export function KuiButton({ onClick, isDisabled }) {
+  return <button className="kuiButton" onClick={onClick} isDisabled={isDisabled}/>
+};
+```
+Bad:
+```
+export class KuiButton extends React.Component {
+  render({ onClick, isDisabled }) {
```

Comment:
I think the `render()` functions of class-style components don't receive their props as arguments. They are available as `this.props` though.